libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/RestClientTest.java

```diff
@@ -588,7 +588,20 @@ public void testDoubleConsume() throws Exception {
         	fail("Calling consume should not have thrown an exception");
         }
     }
-    
+
+    /**
+     * Testing a search call to the server.
+     * Create new account then look for it using sosl.
```

Comment:
Could we remove that line from the comment

Reply:
Sure.